### ISSUE TITLE
Availability of talos_data?

### ISSUE THREAD

**Issue by wxmerkt** [nmansard/wsmemmo_pinocchio#1]
It is not on `robotpkg`. Since `talos_description` is now released via ROS to `kinetic` and `melodic`, could just the `talos_left_arm.urdf` be added to this repository? The meshes could then be retrieved from the ROS-available description packages.

**Comment by wxmerkt** [comment]
Workaround:

1. Download http://robotpkg.openrobots.org/distfiles/talos_data-0.0.20.tar.gz
2. Create the folder: `sudo mkdir -p /opt/openrobots/share`
3. Move the extracted archive to `/opt/openrobots/share/talos_data`: `sudo mv talos_data-0.0.20 /opt/openrobots/share/talos_data`

**Comment by olivier-stasse** [comment]
Dear Wolfgang,
Since January 17th of January talos_robot has become LGPL-v3.0.
It took us a while to commute but now you can now have talos_data here:
 * https://github.com/stack-of-tasks/talos_data
 * and through robotpkg using version 0.0.21

Best,
Olivier.

**Comment by wxmerkt** [comment]
Thank you Olivier! This is a really helpful link. We had some trouble installing via robotpkg during the Winter School and used the above workaround.

**Comment by olivier-stasse** [comment]
Yes sorry we were over conservative and the previous license needed a password to have access to talos-data. Thanks to PAL-Robotics we could switch to a public release.
Let me know if you have problem with robotpkg, this is our own ppa tested over various free OS. So we do our best to have it work on 16.04 LTS. We also have regression test on 18.04 LTS but with PAL we agreed that Trusty and Kinetic will be on main target during the project. 

If you need 18.04 LTS we can only try to work and guarantee the software from our group.

**Comment by wxmerkt** [comment]
We are happy to have it in a source workspace, just didn't know where to look. 

Also, `talos_description` is now released directly to Kinetic and Melodic thanks to PAL and this is great to have for planning :). The difference is thus mostly the availability of the special URDFs (arm, legs, etc.)
